creation/views.py

```diff
@@ -2861,3 +2943,750 @@ def update_codefiles(request):
     }
     context.update(csrf(request))
     return render(request, 'creation/templates/update_codefiles.html', context)
+
+# -------------- Bidding Module -----------------
+
+@login_required
+def rate_contributors(request, sel_status):
+    
+    # Check if Language Manager for this request
+    if not is_language_manager(request.user):
+        raise PermissionDenied()
+
+    lang_select = request.GET.get('language')
+    if lang_select == '':
+        return HttpResponseRedirect(request.META['HTTP_REFERER'])
+    elif is_language_manager(request.user):
```

Comment:
elif not needed here!
You have already checked at the top whether the user is language manager or not!